engine-tests/src/main/java/org/terasology/testUtil/ModuleManagerFactory.java

```diff
@@ -35,6 +36,15 @@ private ModuleManagerFactory() {
 
     public static ModuleManager create() throws Exception {
         ModuleManager moduleManager = new ModuleManagerImpl("");
+        return LoadModuleEnvironment(moduleManager);
+    }
+
+    public static ModuleManager create(String masterServerAddress) throws Exception {
+        ModuleManager moduleManager = new ModuleManagerImpl(masterServerAddress);
+        return LoadModuleEnvironment(moduleManager);
+    }
+
+    private static ModuleManager LoadModuleEnvironment(ModuleManager moduleManager) throws Exception {
```

Comment:
This is a method and thus should start with a lower case letter.
```suggestion
    private static ModuleManager loadModuleEnvironment(ModuleManager moduleManager) throws Exception {
```

Reply:
applied lowerCamelCase